config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```
@@ -495,3 +495,36 @@ periodics:
       - --test_args=--ginkgo.focus=\[sig-network\]\sServices --ginkgo.skip=\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:GCEAlphaFeature\] --minStartupPods=8
       - --timeout=300m
       image: gcr.io/k8s-testimages/kubekins-e2e:v20200306-3e08456-master
+- interval: 12h
+  name: ci-basic-sctp-e2e
```

The job names live in a global namespace so they need to be more informative
```suggestion
  name: ci-kubernetes-e2e-gci-gke-sctp
```

---

Fixed. Now it has a GCE "namespaced" name, just like to lb-finalizer above this one.